examples/tests/sdk_tests/regex/regexp.test.w

```diff
@@ -0,0 +1,31 @@
+bring regex;
+bring expect;
+
+let r: regex.Regexp = regex.compile("p([a-z]+)ch");
```

Comment:
Can we make this a built-in type like the built-in `datetime` and `duration` types? I feel like regular expressions are probably handy enough to make it included without needing a `bring`.

Internally the TypeScript class can be named `std.RegExp` but some logic can be added to the compiler so that users only have to type `regexp`. If you grep for "datetime" in this file you'll see a few places where there's code for mapping types like "std.Datetime" into "datetime". We'll want to do the same thing so that Wing will translate code like this:

```js
let r: regexp = regexp.compile("p([a-z]+)ch");
```

into this, under the hood:
```js
let r: std.RegExp = std.RegExp.compile("p([a-z]+)ch");
```

We can then get rid of the `regex` module and things will be a tiny bit cleaner 🧼

Reply:
I'm getting this after applying the changes:

```sh
error: Unknown symbol "regex"
```